menu.yaml

```diff
@@ -326,4 +326,4 @@
   - panels/json/gitter-index-pattern.json
   menu:
   - name: Overview
-    panel: panels/json/gitter.json
\ No newline at end of file
+    panel: panels/json/gitter.json
```

Comment:
Just in case, this line is marked as changed but it was not modified.

Reply:
I think it is because the change is he added a new empty line (330) after 329. Previously it was not there.

Reply:
You are right @vchrombie, thank you!